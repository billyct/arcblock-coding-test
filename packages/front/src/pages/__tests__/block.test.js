import {render} from '@testing-library/react'

jest.mock('../../hooks/useBlockRouteParams')
const useBlockRouteParams = require('../../hooks/useBlockRouteParams')

jest.mock('../../components/Block', () => 'block-component')
jest.mock('../../components/Search', () => 'search-component')

afterAll(() => {
  jest.resetModules()
})

test('BlockPage Component', () => {
  useBlockRouteParams.__setMockData({})

  const BlockPage = require('../block').default
  const {container} = render(<BlockPage/>)
  expect(container).toMatchSnapshot()
})
