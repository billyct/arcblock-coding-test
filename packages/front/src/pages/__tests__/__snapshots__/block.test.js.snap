// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockPage Component 1`] = `
<div>
  <div
    class="bg-gray-50 min-h-screen pb-10"
  >
    <header
      class="p-4 shadow-sm border-b border-gray-200 bg-white mb-4"
    >
      <div
        class="container mx-auto flex items-center "
      >
        <a
          href="/"
        >
          <svg
            class="cursor-pointer"
            height="36"
            width="36"
          >
            logo.svg
          </svg>
        </a>
        <search-component
          classname="flex-1 ml-4"
        />
      </div>
    </header>
    <div
      class="px-4"
    >
      <div
        class="container mx-auto"
      >
        <block-component />
      </div>
    </div>
  </div>
</div>
`;
